### DISCUSSION TITLE
where is Smooth Locomotion ??

### DISCUSSION THREAD

**Discussion by ragestaker21** [praydog/REFramework#907] [General Questions] [answered]
on the Smooth Locomotion  mod page it says reframework has it integrated but when i play and i move forward there is head wobbling each step the character takes, how do i enable Smooth Locomotion ?

**Comment by praydog** [comment]
Nightly builds https://github.com/praydog/REFramework-nightly/releases/

**Reply by ragestaker21** [reply]
yes im already using nightly where do i enable Smooth Locomotion

**Reply by praydog** [reply, marked as answer]
It's in it by default in one of the scripts that comes with it, just doesn't work with mouse and keyboard

**Reply by ragestaker21** [reply]
:(
